src/components/Slider/utils.ts

```diff
@@ -88,7 +88,7 @@ function createMarks(points: number[]): RcSliderProps['marks'] {
     return marks;
 }
 
-export function getInnerState({
+export function prepareSliderInnerState({
```

Comment:
Could you provide some jsdoc here?

Reply:
Yes, forgot to do it, thanks
https://github.com/gravity-ui/uikit/pull/1637/commits/143584fa5100fdf50b41a55846ab5e66581c9aec